src/openforms/submissions/tests/test_submission_suspension.py

```diff
@@ -262,3 +262,99 @@ def test_identifying_attributes_hashed_on_suspend(self):
         # Assert that identifying attribute is hashed
         self.assertNotEqual(submission.auth_info.value, "123456789")
         self.assertTrue(submission.auth_info.attribute_hashed)
+
```

Comment:
I wonder if these new tests are really needed, since both confirmation/suspension email call the same function `get_wrapper_context` to take care of putting the design tokens in the context? Maybe just a test of `_get_design_token_values` could have been enough?